wpilibj/src/athena/java/edu/wpi/first/wpilibj/ADXRS450_Gyro.java

```diff
@@ -71,6 +74,10 @@ public ADXRS450_Gyro(SPI.Port port) {
           false);
       return;
     }
+    
+    // Grab Accumulator Handle
+    m_gyroHandle = GyroJNI.initializeSPIGyro((byte)port.getValue());
+    
 
     m_spi.initAccumulator(kSamplePeriod, 0x20000000, 4, 0x0c00000e, 0x04000000,
```

Comment:
Thats a good point, as long as we are sure nobody will want to use those for a future purpose. I'll split the implementation files at that point as well, as the ifs would get way too complicated at that point.

The only change is teams would not be able to change the calibrate methods themselves. But thats probably not the end of the world. I would like to leave the old SPIAccumulator functions in, as I could see some teams wanting to be able to use a different gyro, which would need raw access.